lockbox-iosTests/AccountSettingPresenterSpec.swift

```diff
@@ -163,6 +172,22 @@ class AccountSettingPresenterSpec: QuickSpec {
                 }
             }
 
+            describe("unlinkAccountTapped") {
```

Comment:
This test looks good; rather than calling `unlinkAccountObserver` manually in the previous test, you can call `onNext` on the observer saved in your `displayAlertActionButtons` list to simulate the `AlertAction` tap.

Reply:
Done